src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=acaddemicts
 spring.datasource.password=acaddemicts
-spring.jpa.hibernate.ddl-auto=create-drop
-spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
It should stay at create-drop I think, or validate

Reply:
Fixed